### ISSUE TITLE
About the training of Encoder: approx. training time needed to successfully reconstruct the real image

### ISSUE THREAD

**Issue by yccyenchicheng** [genforce/idinvert#16]
Thank you for your great work!

I am training the encoder for my own dataset with the default hyperparameters.

However, in the early stage (~160 kimgs), the adv loss seems to dominate the recon loss (the reconstructed images are meaningful but related to the encoded images).

I would like to know what is the training time needed (or trained kimgs needed) to see the reconstructed images to start to look similar to the encoded images, empirically?

If not, what is the possible issue to cause that?

Thank you!

**Comment by zhujiapeng** [comment]
You could add a batch norm layer after each convolution layer in the encoder, this might works.

**Comment by yccyenchicheng** [comment]
Thank you so much!
